.github/workflows/MainDistributionPipeline.yml

```
@@ -18,12 +18,12 @@ jobs:
-      extension_name: quack
+      extension_name: fast_lanes
 
   duckdb-stable-build:
     name: Build extension binaries
     uses: duckdb/extension-ci-tools/.github/workflows/_extension_distribution.yml@v1.2.1
     with:
       duckdb_version: v1.2.1
       ci_tools_version: v1.2.1
-      extension_name: quack
\ No newline at end of file
+      extension_name: fast_lanes
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)